pkg/commands/image/patch_test.go

```diff
@@ -438,7 +442,9 @@ status: {}
 
 		when("output flag is used", func() {
 			it("does not patch and prints the resource output", func() {
```

Comment:
yes, indeed... ugh 🙅 . because the image gets updated only if patch happens and patch does not happen when you do --dry-run.
please let me know, if you have any ideas on how to handle this.

Reply:
[outputting the patch file (instead of resource yaml)](https://github.com/vmware-tanzu/kpack-cli/pull/91#discussion_r499801448) would fix things 😬 . The output would not differ between --dry-run or otherwise, which is probably not a big deal. looking at alternatives before going down this path.

Reply:
all good. WIP

Reply:
marking as resolved. addressed in commit https://github.com/vmware-tanzu/kpack-cli/pull/91/commits/0a6deb8b4bef472550c99e327104b4e65566a391